sources/packages/backend/apps/api/src/route-controllers/student-appeal/student-appeal.institutions.controller.ts

```diff
@@ -0,0 +1,49 @@
+import { Controller, Param, Get, ParseIntPipe } from "@nestjs/common";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { 
+  AllowAuthorizedParty,
+  IsBCPublicInstitution,
+  HasStudentDataAccess, 
+} from "../../auth/decorators";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import BaseController from "../BaseController";
+import { ClientTypeBaseRoute } from "../../types";
+import { DetailedStudentAppealAPIOutDTO } from "./models/student-appeal.dto";
+import { StudentAppealControllerService } from "./student-appeal.controller.service";
+
+/**
+ * Student appeal controller for institutions.
+ */
+@AllowAuthorizedParty(AuthorizedParties.institution)
+@IsBCPublicInstitution()
+@Controller("appeal")
+@ApiTags(`${ClientTypeBaseRoute.Institution}-appeal`)
+export class StudentAppealInstitutionsController extends BaseController {
+  constructor(
+    private readonly studentAppealControllerService: StudentAppealControllerService,
+  ) {
+    super();
+  }
+
+  /**
+   * Get the student appeal details.
+   * @param appealId appeal id to be retrieved.
```

Comment:
The order of comments are reversed.

Reply:
are we following the order of params in the comments everywhere else 😄 . 

updated